scripts/systemd/nutshutdown.in

```diff
@@ -1,2 +1,16 @@
 #!/bin/sh
-@SBINDIR@/upsmon -K >/dev/null 2>&1 && @SBINDIR@/upsdrvctl shutdown
+
+if @SBINDIR@/upsmon -K >/dev/null 2>&1; then
+  wait_delay=`/bin/sed -ne 's#^ *POWEROFF_WAIT= *\(.*\)$#\1#p' @sysconfdir@/nut.conf`
```

Comment:
Yes, those `${prefix}` verbatims are evil and hard to avoid, struck a few more of those in the system/SMF instance wrapping PR ;)